Sources/Subs.php

```diff
@@ -7040,4 +7040,78 @@ function sentence_list($list)
 	return strtr($format, $replacements);
 }
 
+/**
+ * Truncate an array to a specified length
+ *
+ * @param array $array The array to truncate
+ * @param int $max_length The upperbound on the length
+ * @param int $deep How levels in an multidimensional array should the function take into account.
+ * @return array The truncated array
+ */
+function truncate_array($array, $max_length = 1900, $deep = 3)
```

Comment:
why not camel case?

Reply:
We don't have a proper set of rules/guides for this kind of stuff but we should and for now it will be better to follow what we already have.

Reply:
Yeah, officially we accept both camel case and underscore separation, but the unofficial preference has been for underscore separation for a while now. Perhaps we should update the official policy to make that preference clear.